include/inviwo/core/util/pathtype.h

```diff
@@ -42,6 +42,7 @@ enum class PathType {
     Resources,          // /resources
     TransferFunctions,  // /data/transferfunctions
     Settings,           // path to the current users settings
+    Cache,              // cache folder in the current users settings
```

Comment:
I think that the fetched data should be in the downloads folder, at least on windows. This should in that case be “Downloads” instead
```suggestion
    Downloads,              // current user’s download directory 
```

Reply:
A application should **never** write in to a users files, be in downloads, documents etc, without explicitly asking about it each and every time. 

Caching like this is usually done to some "hidden" directly, consider how chrome does it caching, or vcpkg. or basically any other software. 

I write in the log that the file was cached and to where so, it is not impossible for a user to find. And if we wan we can add some option to clear the cache.

Reply:
I thought that it was more like downloading something from an application, but I don't have strong opinions about it.